src/test/cs/Infinispan/HotRod/RemoteEventTest.cs

```diff
@@ -88,6 +96,25 @@ public void CreatedEventTest()
         }
 
         [Test]
+        public void CustomEventTest()
```

Comment:
This test is not available in https://github.com/infinispan/dotnet-client/pull/107 . Is it supposed to be here or not? My intention was to review this one and close 107 automatically as all the commits are included in the current one.